### PR TITLE
Deprecate google.cloud

### PR DESCRIPTION
**Pull request by mariolenz**
Deprecate `google.cloud` as discussed in ansible-community/community-topics#105 and voted on in ansible-community/community-topics#121

### CODE REVIEW

File: 6/changelog.yaml
Comment (felixfontein) [inline 6/changelog.yaml:93]:
```suggestion
        removed from Ansible 8
```

**Review by felixfontein — APPROVED** [review]

File: 6/changelog.yaml
Comment (felixfontein) [inline 6/changelog.yaml:93]: Hmm, I'm wondering whether we should expand this to explain that this is not a final decision, but just part of the process (https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#cancelling-removal-of-an-unmaintained-collection).

Comment (mariolenz) [reply 6/changelog.yaml:93]: @felixfontein I was pondering myself if we should do this. How about `will be removed from Ansible 8 if nobody starts maintaining it again until then`? This makes clear that it's not a final decision.

I'm not sure if we should mention the exact process here, though.

Comment (mariolenz) [reply 6/changelog.yaml]: @felixfontein What do you think about this / my last commit?

File: 6/changelog.yaml
Comment (felixfontein) [inline 6/changelog.yaml:93]: I guess it's best to also link to the process for details:
```suggestion
        removed from Ansible 8 if nobody starts maintaining it again until then.
        See `the removal process for details on how this works
        <https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#cancelling-removal-of-an-unmaintained-collection>`__
```

File: 6/changelog.yaml
Comment (felixfontein) [reply 6/changelog.yaml]: Looks good! I've added a suggestion with a link to the process description.

**Review by felixfontein — APPROVED** [review]
Looks good IMO!

**Comment by mariolenz** [comment]
> Looks good IMO!

OK, how do we get this merged? Do you do it directly, or do we have to add a `mergeit`or `shipit` or do we need a second review from someone else..?

File: 6/changelog.yaml
Comment (briantist) [inline 6/changelog.yaml:95]: I hadn't seen the double underscore before and wanted to know it was a typo, but it is not! TIL
https://stackoverflow.com/a/27435441/3905079

**Review by briantist — APPROVED** [review]

File: 6/changelog.yaml
Comment (samccann) [inline 6/changelog.yaml:93]:
```suggestion
        removed from Ansible 8 if no one starts maintaining it again before Ansible 8.
```

File: 6/changelog.yaml
Comment (samccann) [reply 6/changelog.yaml:95]: omgosh ditto!  TIL... and I think I've seen some of those duplicate target name warnings in the docs build and never figured out what was wrong... :-)

Comment (felixfontein) [reply 6/changelog.yaml:95]: I'm expecting that in the future we'll copy'n'paste this fragment, so the double underscore will prevent that warning then :-)

**Comment by mariolenz** [comment]
@samccann I've committed your suggestion. Could you please review again?

@briantist @felixfontein I didn't request another review from you, but feel free to have another look at this PR and mention anything you think should be changed.

**Review by samccann — APPROVED** [review]

**Review by felixfontein — APPROVED** [review]
LGTM! I think we can also use the same text for the Bullhorn announcement.

**Comment by mariolenz** [comment]
> LGTM! I think we can also use the same text for the Bullhorn announcement.

@felixfontein Can I announce this on Bullhorn yet, or should I wait until this PR is merged?

**Comment by felixfontein** [comment]
@mariolenz @briantist @samccann thanks a lot for creating and reviewing this!

@mariolenz sure, feel free to do that!

**Pull request merged**
